modules/ob-build-secret-annotation.adoc

```diff
@@ -0,0 +1,25 @@
+// This module is included in the following assembly:
+//
+// builds/authenticating-build-images.adoc
+
+:_content-type: CONCEPT
+[id="ob-build-secret-annotation_{context}"]
+= Build secret annotation
+
+You can add an annotation `build.shipwright.io/referenced.secret: "true"` to a build secret. Based on this annotation, the build controller takes a reconcile action when an event, such as create, update, or delete triggers for the build secret. The following example shows the usage of an annotation with a secret:
+
+[source,yaml]
+----
+apiVersion: v1
+data:
+  .dockerconfigjson: xxxxx
```

Comment:
`xxxxx` should be styled as a user-replaced value in some way. For example, another spot in the repo uses:

```
data:
  .dockerconfigjson: <pull_secret> <4>
```
and then
```
<4> Base64-encoded pull secret
```

Reply:
@apoorvajagtap Can we use the above suggested user-replaced value?

Reply:
Yes, that works.
The focus of this context is more on `metadata.annotations` section, and I believe that was the reason for no details highlighted in the `data`.